src/Okta.Idx.Sdk/SignInWidgetConfiguration.cs

```diff
@@ -17,7 +17,7 @@ public class SignInWidgetConfiguration
         /// <summary>
         /// The default version.
         /// </summary>
-        public const string DefaultVersion = "5.5.2";
+        public const string DefaultVersion = "5.8.0";
```

Comment:
Do we still need this constant? I noticed we're explicitly setting the version number in the head.

Reply:
I changed it in both places to hopefully get as many consumers as possible.  The default is used if no value is specifically specified.